snapshots/papers/2023-06-07/commodity_prices.xlsx.dvc

```diff
@@ -0,0 +1,24 @@
+meta:
+  namespace: papers
+  short_name: commodity_prices
+  name: Real Commodity Price Index since 1850 - Jacks (2016)
+  version: '2023-06-07'
+  publication_year: 2020
+  source_name: 'Jacks, D.S. (2019), "From Boom to Bust: A Typology of Real Commodity
+    Prices in the Long Run." Cliometrica 13(2), 202-220.'
+  source_published_by: 'Jacks, D.S. (2019), "From Boom to Bust: A Typology of Real
+    Commodity Prices in the Long Run." Cliometrica 13(2), 202-220.'
+  url: http://www.sfu.ca/~djacks/data/boombust/index.html
+  source_data_url: 
+    http://www.sfu.ca/~djacks/data/boombust/Real%20commodity%20prices,%201850-2020.xlsx
+  file_extension: xlsx
+  date_accessed: 2023-06-07
+  is_public: true
```

Comment:
I'd add as license either "Public domain" or "CC BY" (given that the author says "These data are freely provided to all comers, and questions and/or comments are welcome.
At the same time, I ask that if you use any of these files that you cite the following: ..."). You can use that link as the license_url:
https://www.sfu.ca/~djacks/data/boombust/index.html

Reply:
Thanks! I'm new to licensing.